src/app/app.module.ts

```diff
@@ -7,6 +7,7 @@ import { HomeComponent } from './home/home.component';
 import { AboutComponent } from './about/about.component';
 import { ContactComponent } from './contact/contact.component';
 import { ProductComponent } from './product/product.component';
+import { RouterModule } from '@angular/router';
```

Comment:
please check the syntax

Reply:
thank you. xxx

Reply:
Please check again.
commit 17f55c4c1cfe161aaed935070d335666a4f1932d